src/app/model/ChessEngine.ts

```diff
@@ -2,15 +2,15 @@ import { ChessModel } from './ChessModel';
 import { Side, Square } from './Types';
 import { Piece } from './Piece';
 import { Player } from './Player';
-import { ChessBoard } from './ChessBoard';
+import { Chessboard } from './Chessboard';
```

Comment:
Czy ta zmiana należała do tego PR?

Reply:
Nie, ale już za dużo zmian poszło u mnie, żebym to teraz cofam ... ale to chyba super problematyczne nie powinno być :-)